lib/scalingo/config.rb

```diff
@@ -14,6 +14,7 @@ module Scalingo
   # Endpoints URLS
   setting :urls do
     setting :auth, "https://auth.scalingo.com/v1"
+    setting :billing, "https://cashmachine.scalingo.com/"
```

Comment:
We might want to change that to billing.scalingo.com, this is not a nice URL for public use :/ 

Issue opened here: https://github.com/Scalingo/scalingo-terraform/issues/120

Reply:
Yes indeed. I'll follow that one